dist/stat_data.py

```diff
@@ -159,6 +159,7 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     ConnStat('cond_auto_wait', 'auto adjusting condition wait calls'),
     ConnStat('cond_auto_wait_reset', 'auto adjusting condition resets'),
+    ConnStat('cond_wait_adjust_failed', 'adjusting condition wait failed'),
```

Comment:
The text is better as:
`auto adjusting condition wait raced to update timeout and skipped updating`

It's best to prefix match similar statistics so they sort together, and the text you had sounded quite alarming to me, I don't think it needs to be.

Reply:
Fixed as you suggested.